portfolio/src/main/webapp/script.js

```diff
@@ -73,4 +73,10 @@ function addRandomProject() {
   linkElement = document.getElementById('link-container');
   linkElement.text = link.description;
   linkElement.href = link.url;
-}
\ No newline at end of file
+}
+
+function appendData() {
```

Comment:
Can you add some documentation here about what this is expected to do.  Also possibly change the name to something a little more descriptive?